pkg/services/alertmanager.go

```diff
@@ -0,0 +1,207 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	texttemplate "text/template"
+	"time"
+
+	httputil "github.com/argoproj/notifications-engine/pkg/util/http"
```

Comment:
Would you fix it that golangci-lint has error.

Reply:
I updated the local version of golangci and found this problem. I will fix it and add request contents test case.